homework6.2/homework6.2.py

```diff
@@ -0,0 +1,8 @@
+while True:
```

Comment:
Where is task description? Please check your code even if this is two lines with linter